webcurator-core/src/test/java/org/webcurator/core/store/DigitalAssetStoreClientTest.java

```diff
@@ -70,6 +71,29 @@ public void testSaveFileViaStream() {
         }
     }
 
+    @Test
+    public void testSaveLargeFileViaStream() {
+        RestTemplateBuilder restTemplateBuilder = new RestTemplateBuilder();
+        DigitalAssetStoreClient dasClient = new DigitalAssetStoreClient("http", "localhost", 8082, restTemplateBuilder);
+        dasClient.setFileUploadMode("stream");
+        dasClient.setHarvestBaseUrl("http://localhost:8081");
+
+        File arcDir = new File("/home/leefr/tmp/1122/1");
+        assertNotNull(arcDir);
+
+        File[] arcFiles = arcDir.listFiles();
+        assertNotNull(arcFiles);
+
+        for (File arcFile : arcFiles) {
+            try {
+                dasClient.save("1122", "1", arcFile.toPath());
+            } catch (DigitalAssetStoreException e) {
+                e.printStackTrace();
+                assert false;
+            }
```

Comment:
@leefrank9527 this test fails in my local environment, due to not having the directory of files in `/home/leefr/tmp/1122/1`.

How do you want to handle this test in the codebase? I don't think we should store large test files in the codebase just for this test. Perhaps we could annotate the test as `@ignore` , and leave it there for specific manually testing as required. What do you think?

Reply:
Yes. This test case should be marked as @ignore. Such a big test file is not suitable be contained in the project,  and it requires the Store and Harvest Agent  are running. It's meaningless to test with mocked instances.